Remmber#6.4.py

```diff
@@ -0,0 +1,17 @@
+import imageio as iio
+
+# read an image

```

Comment:
Separate code to small functions- 
https://softwareengineering.stackexchange.com/questions/308108/when-is-it-appropriate-to-make-a-separate-function-when-there-will-only-ever-be